collection/collection_map.go

```diff
@@ -147,10 +147,9 @@ func (c *Map) AddUnique(key string, value string) {
 // internally converts [] string to []types.AnchoredVar
 // with case sensitive vKey
 func (c *Map) SetCS(key string, vKey string, values []string) {
-	c.data[key] = []types.AnchoredVar{}
+	c.data[key] = make([]types.AnchoredVar, 0, len(values))
```

Comment:
I would go ahead and use `make([], len)` and assignment instead of append if we're preallocating